### PR TITLE
[runtime] fix trace handler

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Checkout the rolling builds.

**Comment by ghost** [comment]
@isuruf Look okay?

File: CMakeLists.txt [hunk 1]
```diff
@@ -342,7 +342,8 @@ macro(add_flang_library name)
   endif( LLVM_COMMON_DEPENDS )
 
   llvm_config( ${name} ${LLVM_LINK_COMPONENTS} )
-  target_link_libraries( ${name} ${LLVM_COMMON_LIBS} )
+  list(APPEND LLVM_COMMON_LIBS Dbghelp.lib)
```
Comment (isuruf) [inline CMakeLists.txt]: This should be included in windows only.

Comment (ghost) [reply CMakeLists.txt]: Good catch.

**Review by isuruf — APPROVED** [review]

**Pull request merged**